src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -292,11 +315,14 @@ public String toString() {
                + sessionVisibleFromTime + ", resultsVisibleFromTime="
                + resultsVisibleFromTime + ", timeZone=" + timeZone
                + ", gracePeriod=" + getGracePeriodMinutes() + "min"
+               + ", sentOpeningSoonEmail=" + sentOpeningSoonEmail
                + ", sentOpenEmail=" + sentOpenEmail
                + ", sentPublishedEmail=" + sentPublishedEmail
                + ", isOpeningEmailEnabled=" + isOpeningEmailEnabled
                + ", isClosingEmailEnabled=" + isClosingEmailEnabled
-               + ", isPublishedEmailEnabled=" + isPublishedEmailEnabled + "]";
+               + ", isPublishedEmailEnabled=" + isPublishedEmailEnabled
+               + "]";
+        // todo should sentOpeningSoonEmail be added ? why are some attributes not present here (sentClosingEmail etc)
```

Comment:
yes all attrs should probably be added here. I think I was the one doing the sending closing email PR, and forgot to add it in

Reply:
Ah i see, I've added `sentClosingEmail, sentClosedEmail, createdTime, deletedTime` to the toString now. If any of them aren't needed I can remove them